pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -655,6 +655,8 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
       nativeBuildInputs = with pkgsBuildBuild.python3.pkgs; [ sphinxHook python-docs-theme ];
     };
 
+    availableOn = platform: !platform.isStatic && lib.meta.availableOn platform self;
```

Comment:
Python can be built statically.  Even if it didn't, we could encode that using `badPlatforms = [ lib.systems.inspect.platformPatterns.isStatic ]`, with no need to invent a new helper.